apps/frontend/src/libs/components/header/header.tsx

```diff
@@ -13,15 +13,17 @@ const Header = (): JSX.Element => {
 		return <></>;
 	}
 
-	const userName = authenticatedUser.name;
+	const { email, name } = authenticatedUser;
 
 	return (
 		<header className={styles["header"]}>
 			<div className={styles["logo-container"]}>
 				<img alt="GitFit logo" className={styles["logo-img"]} src={logoSrc} />
 				<span className={styles["logo-text"]}>Logo</span>
 			</div>
-			<Avatar name={userName} />
+			<Popover content={<UserPopover email={email} name={name} />}>
+				<Avatar name={name} />
+			</Popover>
```

Comment:
This looks a bit ugly (`content` prop) and confusing(`UserPopover` as a content). 
Reading `UserPopover` I expected it to be `Popover` wrapper. So I expect something like:
```suggestion
			<UserPopover email={email} name={name}>
				<Avatar name={name} />
			</UserPopover >
```
where `UserPopover` will use `Popover` behind the scene

Reply:
Agree, I'll try to improve it.